src/components/MilestoneConversationItem.jsx

```diff
@@ -85,7 +85,7 @@ const getReadableMessageContext = conversation => {
           </React.Fragment>
         );
       }
-      if (donorType === 'dac') {
+      if (donorType === 'giver') {
```

Comment:
It cannot be the solution, we need to fix it in backend to send DAC info. By this change DAC info and User role is not shown in conversation.

Reply:
@aminlatifi 
As we talked I reopened this PR and I made some changes